docs/maui/markup/extensions/semantic-properties.md

```diff
@@ -0,0 +1,49 @@
+---
+title: SemanticProperties extensions - .NET MAUI Community Toolkit
+author: bijington
+description: The SemanticProperties extensions provide a series of extension methods that support the configuring of accessibility related settings.
+ms.date: 09/14/2022
+---
+
+# SemanticProperties extensions
+
+The `SemanticProperties` extensions provide a series of extension methods that support the configuring of accessibility related settings.
+
+Semantic properties are used to define information about which controls should receive accessibility focus and which text should be read aloud to the user. Semantic properties are attached properties that can be added to any element to set the underlying platform accessibility APIs.
+
+For further information on `SemanticProperties` and accessibility please refer to the [.NET MAUI documentation](/dotnet/maui/fundamentals/accessibility).
+
+For information on other accessibility related extensions please refer to the [AutomationProperties extensions](automation-properties.md).
+
+## SemanticDescription
+
+The `SemanticDescription` method sets a short, descriptive string that the platforms screen reader uses to announce the `BindableObject`.
+
+The following example demonstrates how to set the description on a new `Entry`:
+
+```cs
+new Entry().SemanticDescription("Enter your username.");
+```
+
+## SemanticHeadingLevel
+
+The `SemanticDescription` method sets a heading level to enable the`BindableObject` to be marked as a heading to organize the UI and make it easier to navigate.
```

Comment:
Is this specific to the screen reader? If so, let's add that to the docs.

I'm not sure, but `SemanticHeadingLevel` may also work for tabbing through a list of `Entry`s. If that's the case, we could add some verbiage about that too.

```suggestion
The `SemanticDescription` method sets a heading level to enable the `BindableObject` to be marked as a heading to organize the UI and make it easier to navigate for the platform's screen reader.
```

Reply:
Yeah I like that!